### ISSUE TITLE
Invalid log level

### ISSUE THREAD

**Issue by jonathantneal** [simonsmith/postcss-property-lookup#19]
```bash
/node_modules/postcss-property-lookup/build/index.js:38
    throw new Error('Invalid logLevel: ' + options.logLevel);
          ^
Error: Invalid logLevel: undefined
    at propertyLookup (/node_modules/postcss-property-lookup/build/index.js:38:11)
    at Object.creator [as plugin] (/node_modules/postcss/lib/postcss.js:60:39)
    at /index.js:97:27
    at Array.forEach (native)
    at /index.js:84:13
    at creator (/node_modules/postcss/lib/postcss.js:60:39)
    at Function.postcss.plugin (/node_modules/postcss/lib/postcss.js:66:23)
    at Object.<anonymous> (/index.js:78:26)
    at Module._compile (module.js:460:26)
    at Object.Module._extensions..js (module.js:478:10)
```

I believe the issue may be related to passing an empty object as options to the plugin.

**Comment by jedmao** [comment]
Yeah. You're [overwriting the default options](https://github.com/simonsmith/postcss-property-lookup/blob/0b9fc43b620525c856db12df670e10d0119fc47a/lib/index.js#L11) when you do that.

Why are you passing in an empty object?

**Comment by jonathantneal** [comment]
I am integrating this plugin into PreCSS. Every plugin gets an empty object unless the user appends additional options.

https://github.com/jonathantneal/precss/commit/526e3048f127ecf378f4f120749ad3d954cd8726#diff-168726dbe96b3ce427e7fedce31bb0bcR61

It was recommended to be added https://github.com/jonathantneal/precss/issues/29

**Comment by jedmao** [comment]
Fix published as v1.1.4.
